cpg_workflows/stages/large_cohort.py

```diff
@@ -96,8 +96,8 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
 
         j: PythonJob = get_batch().new_python_job('Combiner', (self.get_job_attrs() or {}) | {'tool': HAIL_QUERY})
         j.image(image_path('cpg_workflows'))
-        j.memory(combiner_config.get('memory', "4Gi"))
-        j.storage(combiner_config.get('storage', "5Gi"))
+        j.memory(combiner_config.get('memory', '4Gi'))
+        j.storage(combiner_config.get('storage', '5Gi'))
```

Comment:
as a side note, any storage request between 0-10Gi will be inflated to 10Gi. That's hail's minimum actual storage if you request any storage